src/HIE/Bios/Internal/Debug.hs

```diff
@@ -21,7 +22,7 @@ import HIE.Bios.Types
 debugInfo :: Cradle
           -> IO String
 debugInfo cradle = unlines <$> do
-    res <- runCradle (cradleOptsProg cradle) (cradleRootDir cradle)
+    res <- getCompilerOptions (cradleRootDir cradle) cradle
```

Comment:
Is there any situation where could not just pass in the cradle and obtain the cradleRootDir in `getCompilerOptions`?

Reply:
Yes, you should pass the path to the file you want to load rather than the root directory. This is hacky.

Reply:
So the `debug` argument should really take a `FilePath` as an argument and use that rather than the root directory.